x-pack/plugins/cases/server/client/mocks.ts

```diff
@@ -83,18 +83,23 @@ export const createCasesClientWithMockSavedObjectsClient = async ({
   const savedObjectsService = savedObjectsServiceMock.createStartContract();
   savedObjectsService.getScopedClient.mockReturnValue(savedObjectsClient);
 
+  // create a fake feature
+  const featureStart = featuresPluginMock.createStart();
+  featureStart.getKibanaFeatures.mockReturnValue([
+    // all the authorization class cares about is the `cases` field in the kibana feature so just cast it to that
+    ({ cases: ['securitySolution'] } as unknown) as KibanaFeature,
```

Comment:
Why we need it?

Reply:
Yeah so if we don't pass in `cases` here, the Authorization class will not find any features and will fail all authorization requests for the tests.

It would hit this `else if` case:

```
    } else if (!isOwnerAvailable) {
      throw Boom.forbidden(this.auditLogger.failure({ owner, operation }));
    }
```

Reply:
I get these errors without the change

```
Summary of all failing tests
 FAIL  x-pack/plugins/cases/server/routes/api/cases/post_case.test.ts (45.347 s)
  ● POST cases › Posts a new case, no connector configured

    expect(received).toEqual(expected) // deep equality

    Expected: 200
    Received: 403

      58 |
      59 |     const response = await routeHandler(context, request, kibanaResponseFactory);
    > 60 |     expect(response.status).toEqual(200);
         |                             ^
      61 |     expect(response.payload.id).toEqual('mock-it');
      62 |     expect(response.payload.status).toEqual('open');
      63 |     expect(response.payload.created_by.username).toEqual('awesome');

      at Object.<anonymous> (x-pack/plugins/cases/server/routes/api/cases/post_case.test.ts:60:29)

  ● POST cases › Posts a new case, connector provided

    expect(received).toEqual(expected) // deep equality

    Expected: 200
    Received: 403

       99 |
      100 |     const response = await routeHandler(context, request, kibanaResponseFactory);
    > 101 |     expect(response.status).toEqual(200);
          |                             ^
      102 |     expect(response.payload.connector).toEqual({
      103 |       id: '123',
      104 |       name: 'Jira',

      at Object.<anonymous> (x-pack/plugins/cases/server/routes/api/cases/post_case.test.ts:101:29)
```

Reply:
I see, thanks!